packages/commerce-sdk-react/src/hooks/useCustomerId.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import useAuth from './useAuth'
+import useLocalStorage from './useLocalStorage'
+
+const onClient = typeof window !== 'undefined'
+
+/**
+ * TBD
+ *
+ */
+const useCustomerId = (): string | null => {
```

Comment:
there is currently no way to get the customer ID for guest/registered user, because the provider manages the token response (including customer id) from SLAS, we need a way to expose the customer id.

we probably also need another hook to expose the access token